testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
So, why do we need to add the @:isvar meta?

And, if we make the var public, what's the point of having getters and setters?